Original test script uses "display" utility which fails without Xserver.
This patch disables this particular test.
Josef Dvoracek | Institute of Physics | Czech Academy of Sciences | 2019-06-13

diff -Nru Healpix_3.50.orig/src/cxx/test/runtest.sh Healpix_3.50/src/cxx/test/runtest.sh
--- Healpix_3.50.orig/src/cxx/test/runtest.sh	2017-03-03 12:30:31.000000000 +0100
+++ Healpix_3.50/src/cxx/test/runtest.sh	2019-06-13 15:45:48.586603000 +0200
@@ -25,4 +25,4 @@
 time $BINPATH/map2tga test_alice3_texture.fits test_alice2.tga -pal 0 -bar -interpol -title "Alice convolved texture" && \
 time $BINPATH/map2tga test_alice3_mod_texture.fits test_alice3.tga -pal 0 -bar -interpol -title "Alice modulated convolved texture"
 
-display test.tga test2.tga test_pw.tga test3.tga test4.tga test5.tga test6.tga test_alice?.tga
+#display test.tga test2.tga test_pw.tga test3.tga test4.tga test5.tga test6.tga test_alice?.tga
